test_scripts/Defects/6_2/2116_OnButtonPress_LimitedHMILevel.lua

```diff
@@ -0,0 +1,64 @@
+---------------------------------------------------------------------------------------------------
+-- https://github.com/smartdevicelink/sdl_core/issues/2116
+-- 
+-- Steps:
+-- 1) Start Core and HMI
+-- 2) Connect mobile app to Core
+-- 3) Mobile app subscribes to OnButtonPress and OnButtonEvent
+-- 4) Mobile app goes to HMI level LIMITED
+-- 5) HMI Sends OnButtonPress and OnButtonEvent specifying the appID of mobile app
+-- 6) Mobile app should receive both OnButtonPress and OnButtonEvent
+-- 7) HMI Sends OnButtonPress and OnButtonEvent without specifying the appID of mobile app
+-- 8) Mobile app should not receive either OnButtonPress or OnButtonEvent
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/TheSameApp/commonTheSameApp')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Data ]]
+local devices = {
+  [1] = { host = "1.0.0.1", port = config.mobilePort }
+}
+
+local appParams = {
+  [1] = { appName = "Test Application", appID = "0001",  fullAppID = "0000001" }
+}
+
+local function receiveButtonPressWithAppId()
+  local mobSession1 = common.mobile.getSession()
+
+  common.hmi.getConnection():SendNotification("Buttons.OnButtonEvent", {name = "OK", mode = "BUTTONDOWN", appID = common.app.getHMIId(1) })
+  mobSession1:ExpectNotification("OnButtonEvent",{buttonName = "OK", buttonEventMode="BUTTONDOWN"}):Times(1)
```

Comment:
```suggestion
  mobSession1:ExpectNotification("OnButtonEvent",{buttonName = "OK", buttonEventMode="BUTTONDOWN"})
```
`Times(1)` isn't necessary. The number of expectations defaults to 1 if `Times` is not specified.